src/Components/Calculator/Calculator.tsx

```diff
@@ -0,0 +1,148 @@
+        }
+    };
+
+    const onKeyClick = (value: number | string) => {
+        if (typeof (value) === "number") onNumberKey(value);
+        else onOperatorKey(value);
+    }
+
+    return (
+        <div className="calculator">
+            <div className="calculator-input">
+                <div className="pre-result">{prevDisplay + operator} </div>
+                <div className="result">{nextValue} </div>
+            </div>
+            <div className="calculator-keypad">
+                {/* First Row */}
+                <CalculatorKey keyValue={"c"} className="calc-btn bg-black" onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={"%"} className="calc-btn bg-black" isDisable={isButtonDisabled} onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={"/"} className="calc-btn bg-black" isDisable={isButtonDisabled} onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={"Back"} className="calc-btn bg-black" onKeyClick={onKeyClick} />
+                {/* Second Row */}
+                <CalculatorKey keyValue={7} className="calc-btn bg-white" onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={8} className="calc-btn bg-white" onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={9} className="calc-btn bg-white" onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={"*"} className="calc-btn bg-black" isDisable={isButtonDisabled} onKeyClick={onKeyClick} />
+                {/* Third Row */}
+                <CalculatorKey keyValue={4} className="calc-btn bg-white" onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={5} className="calc-btn bg-white" onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={6} className="calc-btn bg-white" onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={"-"} className="calc-btn bg-black" isDisable={isButtonDisabled} onKeyClick={onKeyClick} />
+                {/* Fourth Row */}
+                <CalculatorKey keyValue={1} className="calc-btn bg-white" onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={2} className="calc-btn bg-white" onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={3} className="calc-btn bg-white" onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={"+"} className="calc-btn bg-black" isDisable={isButtonDisabled} onKeyClick={onKeyClick} />
+                {/* Fifth Row */}
+                <CalculatorKey keyValue={"\xB1"} className="calc-btn bg-white" isDisable={isButtonDisabled} onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={0} className="key-zero calc-btn bg-white" onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={"."} className="key-dot calc-btn bg-white" isDisable={isButtonDisabled} onKeyClick={onKeyClick} />
+                <CalculatorKey keyValue={"="} className="calc-btn bg-black" isDisable={isButtonDisabled} onKeyClick={onKeyClick} />
```

Comment:
There are a lot of duplicated className here. Why don't you save it as a constant and set it?
It's not good for reusability.
For example, if you wanna use `bg-yellow` instead of `bg-white` here, you should shave more than 20 lines.